controllers/gorch/config_generation.go

```diff
@@ -0,0 +1,599 @@
+package gorch
+
+import (
+	"context"
+	"crypto/sha256"
+	"encoding/hex"
+	"fmt"
+	"github.com/go-logr/logr"
+	"github.com/kserve/kserve/pkg/apis/serving/v1alpha1"
+	kservev1beta1 "github.com/kserve/kserve/pkg/apis/serving/v1beta1"
+	gorchv1alpha1 "github.com/trustyai-explainability/trustyai-service-operator/api/gorch/v1alpha1"
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/apimachinery/pkg/types"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
+	"sort"
+	"strconv"
+	"strings"
+)
+
+// === ORCHESTRATOR CONFIGURATION ======================================================================================
+// defineOrchestratorConfigMap defines ConfigMap *data* for the orchestrator according to the ISVCs+SRs in the namespace
+func (r *GuardrailsOrchestratorReconciler) defineOrchestratorConfigMap(
+	ctx context.Context, orchestrator *gorchv1alpha1.GuardrailsOrchestrator) (*corev1.ConfigMap, []string, error) {
+	log := ctrl.Log.WithName("AutoConfigurator | Orchestrator ConfigMap Definer |")
+	orchestratorName := orchestrator.Name
+	namespace := orchestrator.Namespace
+	generationService := orchestrator.Spec.AutoConfig.InferenceServiceToGuardrail
+	useBuiltInDetectors := orchestrator.Spec.EnableBuiltInDetectors
+	matchLabel := orchestrator.Spec.AutoConfig.DetectorServiceLabelToMatch
+
+	if matchLabel == "" {
+		matchLabel = "trustyai/guardrails"
+	}
+
+	configMapName := orchestratorName + "-auto-config"
+
+	log.Info("Starting automatic orchestrator configmap definition",
+		"orchestratorName", orchestratorName,
+		"namespace", namespace,
+		"generationService", generationService,
+		"useBuiltInDetectors", useBuiltInDetectors,
+	)
+
+	// get generation model
+	var allInferenceServices kservev1beta1.InferenceServiceList
+	err := r.List(ctx, &allInferenceServices, &client.ListOptions{
+		Namespace: namespace,
+	})
+	if err != nil {
+		return nil, nil, fmt.Errorf("Could not list all InferenceServices: %w", err)
```

Comment:
Could the namespace by added `Could not list all InferenceServices in namespace ...`